### PR TITLE
update the ci signal team to kubernetes/release-team-release-signal

### PR DESCRIPTION
**Pull request by pacoxu**
/cc @Vyom-Yadav

### CODE REVIEW

**Comment by Vyom-Yadav** [comment]
> [Keywords](https://help.github.com/articles/closing-issues-using-keywords) which can automatically close issues and at(@) or hashtag(#) mentions are not allowed in commit messages.

@pacoxu Please fix the commit message.

**Comment by Vyom-Yadav** [comment]
@pacoxu The commit message, you changed the title.

File: release-team/role-handbooks/release-signal/README.md [hunk 1]
```diff
@@ -454,15 +455,15 @@ being tracked for this release, the following template is to be used.
...
-    -   [priority/important-soon issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+milestone%3Av1.29+label%3Apriority%2Fimportant-soon+is%3Aissue+sort%3Aupdated-asc+-label%3Akind%2Ffailing-test+-label%3Akind%2Fflake+) (#number)
+- Query: [open non-test/flake k/k issues for v1.30](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+milestone%3Av1.30+is%3Aissue+sort%3Aupdated-asc+-label%3Akind%2Ffailing-test+-label%3Akind%2Fflake+) (#number)
+    -   [priority/critical-urgent issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+milestone%3Av1.30+label%3Apriority%2Fcritical-urgent+is%3Aissue+sort%3Aupdated-asc+-label%3Akind%2Ffailing-test+-label%3Akind%2Fflake+) (#number)
+    -   [priority/important-soon issues](https://github.com/kubernetes/kubernetes/issues?q=is%3Aopen+milestone%3Av1.30+label%3Apriority%2Fimportant-soon+is%3Aissue+sort%3Aupdated-asc+-label%3Akind%2Ffailing-test+-label%3Akind%2Fflake+) (#number)
```
Comment (Vyom-Yadav) [inline release-team/role-handbooks/release-signal/README.md:460]: So we need to make this for every release. I didn't update these as I used them as just examples. But if you feel we need to update these in every cycle, please mention this in the lead handbook.

Same for all other version updates.

**Review by Vyom-Yadav — CHANGES_REQUESTED** [review]
Item:

File: release-team/role-handbooks/release-signal/README.md (hunk 1 above)
Comment (pacoxu) [reply release-team/role-handbooks/release-signal/README.md]: This is something nice to update but not have to as these are just examples. 
I revert this commit.

**Pull request merged**